packages/app/pages/api/updateRedirectionUrl.ts

```diff
@@ -0,0 +1,23 @@
+import { initializeApp } from 'firebase/app';
+import { collection, doc, setDoc, getFirestore } from 'firebase/firestore/lite';
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { firebaseConfig } from '../../components/constants';
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+    switch (req.method) {
+        case "POST":
```

Comment:
nit: `PUT`s are typically idempotent (calling multiple times yields the same result) and `POST`s aren't. bc we are doing a `setDoc` here, it would be idempotent and would traditionally use `PUT` here.

also, we could reuse the same path as the GET single url above